app/modules/listings/components/listing_elements.svelte

```diff
@@ -63,33 +60,41 @@
 
   const _addUriAsElement = async (listingId, entity) => {
     try {
-      const element = await addElement(listingId, entity.uri)
-      if (isNonEmptyArray(element.alreadyInList)) {
+      const { createdElements, alreadyInList } = await addElement(listingId, entity.uri)
+      if (isNonEmptyArray(alreadyInList)) {
         return flash = {
           type: 'info',
           message: i18n('This work is already in the list')
         }
       }
-      entities = [ entity, ...entities ]
+      // Re fetch entities with fitting attributes.
+      await assignEntitiesToElements(createdElements)
+      paginatedElements = [ ...paginatedElements, ...createdElements ]
+      return flash = {
+        type: 'success',
+        message: i18n('Has been added to the list')
```

Comment:
```suggestion
        message: i18n('Added to the list')
```

Reply:
done in 5b3e3bda5f36a66cfe3853fb22c64d7e38a06e60